### DISCUSSION TITLE
If you are in search of star 👇 (Exchange)

### DISCUSSION THREAD

**Discussion by cesarreyes3** [Schweinepriester/github-profile-achievements#241] [Q&A] [answered]
Please leave a star here 🙏🙏 : https://github.com/masbug/flysystem-google-drive-ext
i will help others in search of star,
btw thanks in advance

~ add a star and add leave your repo link here, I will start it

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by cesarreyes3** [reply]
But, you must give one first 😕

**Reply by sulincix** [reply]
done

**Reply by Zaryob** [reply]
done

**Reply by sbsoso0411** [reply]
@number0x0001 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by longquoc47krb** [reply]
Done. https://github.com/longquoc47krb/BookingEventTicket

**Reply by umarwaseeem** [reply]
Kindly leave a star here at a chatting app I made in Dart/Flutter. 
Have a look at the screenshots and documentation too.

https://github.com/Umar-Waseem/Chat-Flutter-Firebase

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@number0x0001 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@Umar-Waseem I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@longquoc47krb I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Chameleon-Coder** [comment, marked as answer]
Done, https://github.com/Chameleon-Coder/Chameleon-Coder

**Reply by cesarreyes3** [reply]
Done

**Reply by sulincix** [reply]
done

**Reply by Zaryob** [reply]
done

**Reply by sbsoso0411** [reply]
@Chameleon-Coder 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by WHAHA-HA** [reply]
@Chameleon-Coder
Already done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by miladkhalighi** [reply]
Done,
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@Chameleon-Coder I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by sulincix** [comment]
Please leave a star here:  https://github.com/sulincix/debian-subsystem

**Reply by cesarreyes3** [reply]
But you didn't give one to me 😕

**Reply by sulincix** [reply]
done

**Reply by cesarreyes3** [reply]
Done

**Reply by cesarreyes3** [reply]
You removed yours, so I remove mine too

**Reply by Zaryob** [reply]
done

**Reply by sbsoso0411** [reply]
@sulincix 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by WHAHA-HA** [reply]
@sulincix 
Already done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@sulincix I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by Zaryob** [comment]
Please leave a star.
https://github.com/Zaryob/inary

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by sbsoso0411** [reply]
@Zaryob 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by WHAHA-HA** [reply]
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@Zaryob I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by lokesh7suman** [comment]
https://github.com/d4rkvent/fonts.git

**Reply by sbsoso0411** [reply]
@d4rkvent 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress thank you

**Reply by WHAHA-HA** [reply]
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by miladkhalighi** [reply]
@d4rkvent done,
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by umarwaseeem** [reply]
Kindly leave a star here at a chatting app I made in Dart/Flutter. 
Have a look at the screenshots and documentation too.

https://github.com/Umar-Waseem/Chat-Flutter-Firebase

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by xommmax** [reply]
Great, I gave it a star! Please star my [Flutter Guidelines](https://github.com/4lfant/flutter-guidelines)

**Comment by zigad** [comment]
Just created a Playground Repo for anyone wishing to play and try new stuff on GitHub without any fear of breaking anything.

Feel free to push, merge, comment, open issues and so on:

https://github.com/zigad/Playground

**Reply by WHAHA-HA** [reply]
@zigad 
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by sbsoso0411** [reply]
@zigad 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by miladkhalighi** [reply]
@zigad done,
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by sbsoso0411** [reply]
@miladkhalighi 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@zigad I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by EeeasyCode** [comment]
https://github.com/EeeasyCode/gachon-discord-crawling-bot
i love you guys
i learn to programing in south-korea
im Beginner 
Look after me plz!!!

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@EeeasyCode I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by sbsoso0411** [comment]
@cesarreyes3 
done, please start my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Comment by WHAHA-HA** [comment]
@cesarreyes3
Already done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by ArielMAJ** [reply]
@WHAHA-HA I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by tanish36** [comment]
Done 
Here my repo : https://github.com/tanish36/BEProject

**Reply by sbsoso0411** [reply]
@tanish36 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by WHAHA-HA** [reply]
@tanish36, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by miladkhalighi** [reply]
@tanish36 done,
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@tanish36 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by miladkhalighi** [comment]
Done
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by sbsoso0411** [reply]
@miladkhalighi 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by miladkhalighi** [reply]
@talent0411 done as well

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by miladkhalighi** [reply]
@kshitij-halankar done

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@miladkhalighi I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by AndPuQing** [comment]


**Reply by sbsoso0411** [reply]
@AndPuQing 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@AndPuQing I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by hasibul442** [comment]
Please Give Star In My Repo 

https://github.com/hasibul442/Online-Crime-Report-System.git

i will do same

**Reply by tanish36** [reply]
done 
Here my repo : https://github.com/tanish36/BEProject

**Reply by hasibul442** [reply]
@tanish36 done

**Reply by sbsoso0411** [reply]
@hasibul442 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by hasibul442** [reply]
@talent0411 done

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by hasibul442** [reply]
@kshitij-halankar done

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@hasibul442 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by longzero2000** [comment]
Random
(https://github.com/longzero2000)

**Reply by sbsoso0411** [reply]
@longzero2000 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by kshitij-halankar** [reply]
Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@longzero2000 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by kshitij-halankar** [comment]
Please give me a star for my repo. I will do the same.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by sbsoso0411** [reply]
@kshitij-halankar 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@kshitij-halankar I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by GabrielPivoto** [comment]


**Reply by sbsoso0411** [reply]
@GabrielPivoto 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by GabrielPivoto** [reply]
@talent0411, done!

**Reply by aarongeo1** [reply]
@GabrielPivoto @talent0411 mine too 
https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Comment by aarongeo1** [comment]
@cesarreyes3 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by sbsoso0411** [reply]
@aarongeo1 
done, please star my repo. https://github.com/fullstack-webdev/Noexam-WordPress

**Reply by aarongeo1** [reply]
i did

**Reply by lokesh7suman** [reply]
done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@aarongeo1 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@sbsoso0411 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by lokesh7suman** [comment]
@aarongeo1 done, please star my repo. https://github.com/d4rkvent/fonts.git

**Reply by ArielMAJ** [reply]
@lokesh7suman I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by bingyun123** [comment]
Done
[https://github.com/bingyun123/custom_chart_viz](https://github.com/bingyun123/custom_chart_viz)

**Reply by ArielMAJ** [reply]
@bingyun123 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by 41pha1** [comment]
Done :)
https://github.com/41pha1/MIDI-Converter

**Comment by ArielMAJ** [comment]
@cesarreyes3 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (MAUI and Winform).

I would appreciate it if you could check it out and give it a star. Thank you.

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
